frontend/libs/studio-components/src/components/StudioError/StudioError.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Alert } from '@digdir/designsystemet-react';
+
+export type StudioErrorProps = {
+  children?: React.ReactNode;
+};
```

Comment:
What about exposing the props from the `Alert` component, but omitting the `severity` prop? 😊 Then we can keep the alert API, but without the severity, which makes sense since this is a `Error` component.

```suggestion
export type StudioErrorProps = Omit<AlertProps, 'severity'>;
```